kihuha/iReporter
# #163185049 - Populate profile incident records

#### What does this PR do?
Consumes all incident records enpoints that are specifically created by the logged in user

#### Description of the task to be completed
Fetch data from the endpoint below and filter out records that the user has created. Populate filtered records to the UI
`GET` `https://ireporter-kihuha.herokuapp.com/api/v2/red-flags`
`GET` `https://ireporter-kihuha.herokuapp.com/api/v2/interventions`

#### How should this be manually tested?
Navigate to `https://kihuha.github.io/iReporter/ui/login` and once logged in, click on the profile button which takes you to the profile page. The incident records displayed should be only those that the logged in user created.

#### Relevant PT stories
https://www.pivotaltracker.com/story/show/163185049
